platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/tokenstore.xml

```diff
@@ -16,7 +16,11 @@
            xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0
         http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
         
-    <bean id="defaultTokenStore" class="ddf.security.sts.DefaultInMemoryTokenStore"
+    <bean id="defaultTokenStore" class="ddf.security.sts.MemoryTokenStore"
           scope="singleton"/>
+
+    <!-- Required for CXF -->
+    <bean id="ehCacheTokenStore" class="ddf.security.sts.DefaultInMemoryTokenStore"
```

Comment:
Since you changed this to use guava should this bean id be renamed?

Reply:
This one is still ehCache. CXF initializes a ehCache token store in the background that isn't used, but needs this code in order for the classloader to have the class.